tyrian/js/src/main/scala/tyrian/TyrianAppF.scala

```diff
@@ -111,14 +111,13 @@ trait TyrianAppF[F[_]: Async, Msg, Model]:
 
   def ready(node: Element, flags: Map[String, String]): Unit =
     run(
-      Tyrian.start(
+      Tyrian.start[F, Model, Msg](
         node,
         router,
         _init(flags),
```

Comment:
I messed up the process somehow here. I thought I was going to suggest a change but I've committed it to your branch via the GH UI - didn't even know that was a thing!

Anyway, you have reverted `_init` to `init` and that was preventing the app from routing when it first opens. So if you were on 'page 2' visited an external link, hit the back button, you'd find yourself on the homepage instead of 'page 2' even though the address bar says `/page-2`.

Reply:
The good news is that with this change, the app loads significantly more quickly and the initial page route happens before the first render, removing an initial flicker than I was going to investigate as a separate issue. :muscle: